examples/3d/split_screen.rs

```diff
@@ -50,81 +50,65 @@ fn setup(
         ..default()
     });
 
-    // Left Camera
-    let left_camera = commands
-        .spawn((
-            Camera3dBundle {
-                transform: Transform::from_xyz(0.0, 200.0, -100.0).looking_at(Vec3::ZERO, Vec3::Y),
-                ..default()
-            },
-            LeftCamera,
-        ))
-        .id();
-
-    // Right Camera
-    let right_camera = commands
-        .spawn((
-            Camera3dBundle {
-                transform: Transform::from_xyz(100.0, 100., 150.0).looking_at(Vec3::ZERO, Vec3::Y),
-                camera: Camera {
-                    // Renders the right camera after the left camera, which has a default priority of 0
-                    order: 1,
-                    // don't clear on the second camera because the first camera already cleared the window
-                    clear_color: ClearColorConfig::None,
-                    ..default()
-                },
-                ..default()
-            },
-            RightCamera,
-        ))
-        .id();
-
-    // Set up UI
-    commands
-        .spawn((
-            TargetCamera(left_camera),
-            NodeBundle {
-                style: Style {
-                    width: Val::Percent(100.),
-                    height: Val::Percent(100.),
-                    ..default()
-                },
-                ..default()
-            },
-        ))
-        .with_children(|parent| {
-            parent.spawn(TextBundle::from_section(
-                "Left",
-                TextStyle {
-                    font_size: 20.,
+    // Cameras and their dedicated UI
+    for (index, (camera_name, camera_pos)) in [
+        ("Player 1", Vec3::new(0.0, 200.0, -150.0)),
+        ("Player 2", Vec3::new(150.0, 150., 50.0)),
+        ("Player 3", Vec3::new(100.0, 150., -150.0)),
+        ("Player 4", Vec3::new(-100.0, 80., 150.0)),
+    ]
+    .iter()
+    .enumerate()
+    {
+        let camera = commands
+            .spawn((
+                Camera3dBundle {
+                    transform: Transform::from_translation(*camera_pos)
+                        .looking_at(Vec3::ZERO, Vec3::Y),
+                    camera: Camera {
+                        // Renders cameras with different priorities to prevent ambiguities
+                        order: index as isize,
+                        // Don't clear after the first camera because the first camera already cleared the entire window
```

Comment:
Is this needed? I thought it would be automatically tracked.

Reply:
At least on the Hanabi example (which is almost the same) this spews a ton of warnings without this.

Reply:
Specifically I mean the clear config. I think default on all cameras will work.